README.markdown

```diff
@@ -731,6 +732,31 @@ class TimeMachine {
 }
 ```
 
+## Objective-C support
+
+`@objc` keyword should be used in a seperate line, on top of entity name, like in provided examples.
+
+```
+@objc
+func performCalculations() { ...
+```
+
+```
+var isMoving: Bool {
+	@objc(setIsMoving:)
```

Comment:
Can you use 4 space intend here? Also having only opening bracket looks weird :)